test/test_accelerator.py

```diff
@@ -0,0 +1,85 @@
+# Owner(s): ["module: tests"]
+
+import sys
+import unittest
+
+import torch
+from torch.testing._internal.common_utils import (
+    NoTest,
+    run_tests,
+    TEST_CUDA,
+    TEST_XPU,
+    TestCase,
+)
+
+
+if not torch.accelerator.is_available():
+    print("No available accelerator detected, skipping tests", file=sys.stderr)
+    TestCase = NoTest  # noqa: F811
+
+
+class TestAccelerator(TestCase):
+    def test_current_accelerator(self):
+        self.assertTrue(torch.accelerator.is_available())
+        accelerators = ["cuda", "xpu"]
```

Comment:
Why hip and mps are not part of the list?

Reply:
`hip` will be covered as it is masqued as `cuda`. And added `mps` to the list.